pandas/tests/indexes/test_numeric.py

```diff
@@ -843,6 +843,28 @@ def test_join_outer(self):
         tm.assert_numpy_array_equal(lidx, elidx)
         tm.assert_numpy_array_equal(ridx, eridx)
 
+    @pytest.mark.parametrize('ind1', [[True] * 5, pd.Index([True] * 5)])
+    @pytest.mark.parametrize('ind2', [[True, False, True, False, False],
```

Comment:
I think you just add these all to TestIndexOps to get overage on all index (and series) for this. Note that there is an open issue to fixture this ;<

Reply:
OK, added some tests to `TestIndexOps`.